src/contracts/PreAMMBatcher.sol

```diff
@@ -405,4 +405,37 @@ contract PreAMMBatcher {
         }
         return newOrders;
     }
+
+    enum Direction {Ascending, Descending}
+
+    function isSortedByLimitPrice(Order[] memory orders, Direction direction)
+        public
```

Comment:
A general comment, it feels icky to make internal methods public just to be able to test them. Maybe there is a way to work around this with the `buidlerevm` that might be worth investigation.

This is a wider problem with Solidity testing in general though and not related to this PR.

Reply:
In the past we have written wrapper contracts that expose these methods as public so they can be tested. I could do that again here. I agree this method should be internal.

Reply:
Adding a new public function increases the gas cost of calling other unrelated functions as well, by the way!

Reply:
I think this could have been done in a separate PR.